co-noir/ultrahonk/src/oink/prover.rs

```diff
@@ -515,12 +521,14 @@ impl<P: HonkCurve<TranscriptFieldType>, H: TranscriptHasher<TranscriptFieldType>
         self.compute_logderivative_inverses(proving_key);
 
         // TACEO TODO: BB does "sparse" commitment here, I don't know if that is necessary (performance wise)
-        Self::commit_to_witness_polynomial(
-            &self.memory.lookup_inverses,
+        let mut lookup_inverses_tmp = std::mem::take(&mut self.memory.lookup_inverses);
```

Comment:
Add a comment on why the take is required (seems to be needed due to double mut self borrow workaround)